### PR TITLE
clear buttons hooked up for clinical submit

### PR DESCRIPTION
**Pull request by jaserud**
**Description of changes**

- gql for clear submission
- added onlcick funcitons for button

**Type of Change**

- [ ] Bug
- [ ] Styling
- [x] New Feature

**Checklist before requesting review:**

- [ ] Matches design:

  - component sizes, spacing, and styles
  - font size, weight, colour
  - spelling has been double checked

- [ ] New uikit components have Storybook stories
- [ ] Feature is minimally responsive.
- [x] Manual testing of UI feature.
- [ ] Selenium test is completed and passing.

### CODE REVIEW

File: components/pages/submission-system/program-clinical-submission/FilesNavigator/index.tsx [hunk 1]
```diff
@@ -10,34 +10,62 @@ import NoData from 'uikit/NoData';
...
+        },
+      });
+      toaster.addToast({
+        variant: 'SUCCESS',
+        interactionType: 'CLOSE',
+        title: 'Cleared',
+        content: `Uploaded ${fileType.toUpperCase()} file has been cleared`,
+      });
+    } catch (err) {
+      commonToaster.unknownErrorWithReloadMessage();
```
Comment (hlminh2000) [inline components/pages/submission-system/program-clinical-submission/FilesNavigator/index.tsx]: the `unknownErrorWithReloadMessage` says it reloaded the data, so in the failure case I think we actually want to do a refetch of the clinical submission query, similar to `handleSubmissionClear` :P

File: components/pages/submission-system/program-clinical-submission/Header.tsx [hunk 2]
```diff
@@ -72,6 +73,16 @@ export default ({
     update: refetchClinicalSubmission,
   });
 
+  const [clearClinicalSubmission] = useMutation<
+    ClinicalSubmissionQueryData,
+    { programShortName: string; submissionVersion: string }
```
Comment (hlminh2000) [inline components/pages/submission-system/program-clinical-submission/Header.tsx]: should this be `ClearSubmissionMutationVariables`?

File: components/pages/submission-system/program-clinical-submission/Header.tsx [hunk 3]
```diff
@@ -188,19 +225,19 @@ export default ({
...
+            <Button
+              variant="text"
+              css={css`
+                margin-right: 10px;
+              `}
+              onClick={handleSubmissionClear}
+            >
+              Clear submission
+            </Button>
+          )}
```
Comment (hlminh2000) [inline components/pages/submission-system/program-clinical-submission/Header.tsx]: thanks for cleaning my mess :P

File: components/pages/submission-system/program-clinical-submission/FilesNavigator/index.tsx (hunk 1 above)
Comment (jaserud) [reply components/pages/submission-system/program-clinical-submission/FilesNavigator/index.tsx]: oops don't know how I missed that :smile:

**Pull request merged**